test/integration/recursor/recursor_spec.rb

```diff
@@ -0,0 +1,27 @@
+Dir['test/libraries/*.rb'].each { |f| require File.expand_path(f) }
```

Comment:
I would be more explicit here about which requires for the library files versus globbing all of them in. Helps expose dependencies more explicitly in my opinion. So a require for each file to me works better than a dir glob trick.